02_state-of-the-art.src.md

```diff
@@ -1,40 +1,40 @@
 # \label{chapter_state_of_the_art}Stand der Technik
 
-In diesem Kapitel werden moderne Anwendungen und ihre Architektur analysiert. Dazu werden zunächst die Begriffe verteilte Systeme und verteilte Dateisysteme definiert. Anschließend werden vier Anwendungen beschrieben, die als Inspiration für das Projekt symCloud verwendet werden.
+In diesem Kapitel werden moderne Anwendungen und ihre Architektur analysiert. Dazu werden zunächst die beiden Begriffe „verteilte Systeme“ und „verteilte Dateisysteme“ definiert. Anschließend werden vier Anwendungen beschrieben, die als Inspiration für das Projekt symCloud verwendet werden.
 
 ## \label{verteilte_systeme}Verteilte Systeme
 
-Andrew Tanenbaum definiert den Begriff der "verteilten Systeme" in seinem Buch folgendermaßen:
+Andrew Tanenbaum definiert den Begriff der „verteilten Systeme" in seinem Buch folgendermaßen:
```

Comment:
nur hoch (probleme beim builden) überprüfen ob das richtig gemacht wird: in der ausgabe zuerst unten dann oben